### DISCUSSION TITLE
Pregunta sobre certificado

### DISCUSSION THREAD

**Discussion by carlosdac55** [rramosp/ai4eng.v1#118] [00 - General]
Buenas noches, Me encuentro desarrollando las actividades del curso, aunque no soy estudiante de la Universidad de Antioquia. Quisiera saber si aún sin estar matriculado en la UdeA tendría un certificado del curso al terminar todas las actividades de este.

Muchas gracias.

**Comment by Felipe-RA** [comment]
Saludos carlosdac55

Sí, en cuanto completes los requisitos y porcentajes expuestos en el sitio del curso podrás descargar el certificado directamente desde la plataforma.
